hack/generator/pkg/astmodel/interface_implementer.go

```diff
@@ -21,9 +21,9 @@ func MakeInterfaceImplementer() InterfaceImplementer {
 	return InterfaceImplementer{}
 }
 
-func (i InterfaceImplementer) HasInterface(name TypeName) bool {
-	_, ok := i.interfaces[name]
-	return ok
+func (i InterfaceImplementer) FindInterface(name TypeName) (*InterfaceImplementation, bool) {
```

Comment:
The `FindInterface()` method is used in PR #1729 when working out which Resources need a test generated. Non-hub resources which implement `astmodel.ConvertibleInterface` need to be tested. Can't detect these by looking for `ConvertTo()` or `ConvertFrom()` because those functions aren't present in `Functions()`; they're encapsulated in the interface implementation.